Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -307,18 +307,31 @@ Overriding event listeners
 --------------------------
 
 Existing event listeners can be overridden by custom implementations. This can be
-performed via :file:`EXT:some_extension/Configuration/Services.yaml`.
+performed with both methods, either by using the PHP :php:`#[AsEventListener]`
+attribute, or via :file:`EXT:some_extension/Configuration/Services.yaml`.
 
 For example, a third-party extension listens on the event
```

Comment:
But it says "by using the PHP #[AsEventListener] attribute" there already, I don't understand?

What exactly should I emphasize how, I'm not sure I understand?

Reply:
For example, a third-party extension listens on the event ... with the following code:"

... code ...

Or via Services.yaml.

-> This does not match IMHO.

So, "For example, a third-party extensions listens on the event via the PHP attribute ..."

... code ...

Or via Services.yaml.

-> This would be better.

Reply:
Emphasize the relevant code, here: the AsEventListener attribute. This makes it easier for readers to catch the relevant part.

Reply:
Added a commit, like this?

Reply:
Do you think it's worth it? Does it add value for the user?
(Wanted to guide you to that feature ;-) ).

Reply:
Yeah, it's pretty nice, I wasn't aware of this :-)